dht.go

```diff
@@ -400,3 +404,27 @@ func (dht *IpfsDHT) protocolStrs() []string {
 func mkDsKey(s string) ds.Key {
 	return ds.NewKey(base32.RawStdEncoding.EncodeToString([]byte(s)))
 }
+
+func (dht *IpfsDHT) PeerID() peer.ID {
```

Comment:
What's the use case for adding these public getter methods? They are nice to have for tests but don't seem essential in the public API.

Reply:
https://github.com/libp2p/go-libp2p-kad-dht/pull/265#discussion_r257440296